Casks/l/luanti.rb

```diff
@@ -0,0 +1,36 @@
+cask "luanti" do
+  arch arm: "arm64", intel: "x86_64"
+
+  version "5.10.0,11.3"
+  sha256 arm:   "aae12650013c1396ade97a7b504039e5615d46d09b6ede47dd5a6a1978cb7479",
+         intel: "60a0e79d686d349c343b14bdadf2fc89ab90b547086d1582fc489c1132c6e461"
+
+  url "https://github.com/minetest/minetest/releases/download/#{version.csv.first}/luanti-#{version.csv.second}-macos#_#{arch}.zip"
+  name "Luanti"
+  desc "Voxel game-creation platform"
+  homepage "https://www.luanti.org/"
+
+  livecheck do
+    url :url
+    regex(/^luanti[._-]v?(\d+(?:\.\d+)+)[._-]macos(\d+(?:\.\d+)+)[._-]#{arch}\.zip$/i)
+    strategy :github_releases do |json, regex|
+      json.map do |release|
+        next if release["draft"] || release["prerelease"]
+
+        release["assets"]&.map do |asset|
+          match = asset["name"]&.match(regex)
+          next if match.blank?
+
+          "#{match[1]},#{match[2]}"
+        end
+      end.flatten
+    end
```

Comment:
```suggestion
    url "https://www.luanti.org/downloads/"
    regex(/href=.*?luanti[._-]v?(\d+(?:\.\d+)+)[._-]macos(\d+(?:\.\d+)+)[._-]#{arch}\.zip/i)
    strategy :page_match do |page, regex|
      page.scan(regex).map { |match| "#{match[0]},#{match[1]}" }
    end
```

We can replace the `GithubReleases` strategy since the first-party download page links to the GitHub release assets.